web-components/src/components/inputs/CheckboxLabel.tsx

```diff
@@ -42,7 +44,7 @@ export default function CheckboxLabel({
             disabled={disabled}
           />
         }
-        label={<Subtitle>{label}</Subtitle>}
+        label={isLabelStringOrNumber ? <Subtitle>{label}</Subtitle> : label}
```

Comment:
Prevent the use of nested typography components.
We were having an error related to that before.